src/core/init/initialize_directfile.ts

```diff
@@ -159,18 +161,26 @@ export default function initializeDirectfileContent({
   const stalled$ = getStalledEvents(mediaElement, clock$)
     .pipe(map(EVENTS.stalled));
 
+  const initiatedEME$ = emeManager$.pipe(
+    filter(({ type }) => !!type.match(/^eme-(init|disabled)$/)),
```

Comment:
Why the RegExp here? It is less readable and I don't see an advantage for that use case